### PR TITLE
ECDC-2622 Refactor stop fw now

### PR DESCRIPTION
**Pull request by kmurica**
Refactoring of code. Decided to keep the function try_send_stop_now to avoid breaking anything in other repositories etc.
Really just a wrapped call of try_send_stop_time with datetime.now() as argument for the stop_time.

### CODE REVIEW

**Review by umithardal — APPROVED** [review]
LGTM. I have tested with CLI. However, @ebadkamil could you test it to see if your data is there with stop now? After that I will merge.

**Comment by umithardal** [comment]
Before merging this refactor, we would like to understand the meaning of `stop_time=0`.

File: file_writer_control/WorkerFinder.py [hunk 1]
```diff
@@ -76,13 +76,7 @@ def try_send_stop_now(self, service_id: str, job_id: str) -> CommandHandler:
...
         :return: A CommandHandler instance for (more) easily checking the outcome of the "stop now" command.
         """
-        command_id = str(uuid.uuid1())
-        message = serialise_stop(
-            job_id=job_id, service_id=service_id, command_id=command_id, stop_time=0
-        )
-        self.command_channel.add_command_id(job_id=job_id, command_id=command_id)
-        self.send_command(message)
-        return CommandHandler(self.command_channel, command_id)
+        return self.try_send_stop_time(service_id, job_id, datetime.now())
```
Comment (ebadkamil) [inline file_writer_control/WorkerFinder.py:79]: why not instead of `datetime.now()` just put 0 so that outcome of this method does not change.

Comment (ebadkamil) [reply file_writer_control/WorkerFinder.py:79]: `return` is not required may be,

Comment (kmurica) [reply file_writer_control/WorkerFinder.py:79]: We don't need the CommandHandler anymore?

Comment (ebadkamil) [reply file_writer_control/WorkerFinder.py:79]: `self.try_send_stop_time` already returns a `CommandHandler`.

Comment (kmurica) [reply file_writer_control/WorkerFinder.py:79]: but the return value will then stay in the scope of the try_send_stop_now function.

Comment (ebadkamil) [reply file_writer_control/WorkerFinder.py:79]: ohh then forget about my comment.

**Review by ebadkamil — APPROVED** [review]

**Pull request merged**